docs/guides/cookie-monitoring.mdx

```diff
@@ -44,7 +44,46 @@ These are end-to-end examples in various programming languages.
 
 <Accordion title="Python SDK Example">
 ```python
-# TODO
+import os
+import time
+
+from flareio import FlareApiClient
+
+
+api_key: str | None = os.environ.get("FLARE_API_KEY")
+if not api_key:
+    raise Exception("Please provide an API key")
+
+api_client = FlareApiClient(api_key=api_key)
+
+last_from: str | None = None
+fetched_pages: int = 0
+
+for resp in api_client.scroll(
+    method="POST",
+    url="/leaksdb/v2/cookies/_search",
+    json={
+        "from": last_from,
+        "domain": "your-domain.com"
+    }
+):
+    # Rate limiting.
+    time.sleep(1)
+
+    resp_data: dict = resp.json()
+
+    fetched_pages += 1
+    num_results: int = len(resp_data["items"])
+    print(f"Fetched page {fetched_pages} with {num_results} results...")
+
+    # Save the last "next" value.
+    last_from = resp_data.get("next") or last_from
+
```

Comment:
Can we add better guidance here? I am not sure "get the full data" is the best explaination ( I know where you copied this from!)

```python
    # Iterate on all cookies
    for cookie in resp_data["items"]:
      if cookie["name"] == "session-token":
        # Invalidate the cookie here
        print(f"Would invalidate cookie {cookie['value']}...")
```

Reply:
I went with `Iterate on cookies in the current page`